core/src/qrsignatures.c

```diff
@@ -0,0 +1,61 @@
+#include <unistd.h>
+#include <stdbool.h>
+#include <string.h>
+#include <stdio.h>
+
+#include "ecdsa.h"
+#include "curves.h"
+#include "memzero.h"
+#include "nist256p1.h"
+#include "qrsignatures.h"
+#include "log.h"
+#include "config.h"
+
+
+
+// 70 is an arbitrary number more than 65(uncompressed key size.)
```

Comment:
suggest to decrease complexity here by declaring the public key as `const uint8_t[65]` as above, and we could remove L16 thru L30